src/animation/AnimationMixer.js

```diff
@@ -83,8 +83,10 @@ Object.assign( THREE.AnimationMixer.prototype, THREE.EventDispatcher.prototype,
 
 		var root = optionalRoot || this._root,
 			rootUuid = root.uuid,
-			clipName = ( typeof clip === 'string' ) ? clip : clip.name,
-			actionsForClip = this._actionsByClip[ clipName ];
+			clipObject = typeof clip === 'string' ? this.getClipByName( clip ) : clip,
```

Comment:
> Am I missing something?

Yep, there's also `stopAllAction` (which is clearly very important) and `update`.

@bhouston 
I guess we can go without the latter and simply have the global time update be global?